### PR TITLE
Docs: Make tables formatting more consistent

### LINKED ISSUE #27821: Remove italic format from tables
**What would you like to be added**:

Some tables have the text in the first column italicized. This is incorrect. Remove that formatting.

**Why is this needed**: Make the documentation formatting more consistent.

### PR DESCRIPTION
**Pull request by jartigag**
**What this PR does / why we need it**:

Fix incorrect formatting in tables and make the documentation formatting more consistent 

**Which issue(s) this PR fixes**:

Fixes #27821

**Special notes for your reviewer**:

### CODE REVIEW

File: docs/sources/datasources/cloudwatch.md [hunk 1]
```diff
@@ -381,7 +381,19 @@ It's now possible to configure data sources using config files with Grafana's pr
 
 Here are some provisioning examples for this data source.
 
-### Using credentials profile name (non-default)
+### Using AWS SDK Default
```
Comment (achatterjee-grafana) [inline docs/sources/datasources/cloudwatch.md]: Using AWS SDK (default)

File: docs/sources/datasources/cloudwatch.md [hunk 2]
```diff
@@ -381,7 +381,19 @@ It's now possible to configure data sources using config files with Grafana's pr
...
+apiVersion: 1
+datasources:
+  - name: CloudWatch
+    type: cloudwatch
+    jsonData:
+      authType: default
+      defaultRegion: eu-west-2
+```
+
+### Using credentials profile name (non-default)
```
Comment (achatterjee-grafana) [inline docs/sources/datasources/cloudwatch.md]: How about "Using profile name of the credential" or "Using credential's profile name"?

**Review by achatterjee-grafana — COMMENTED** [review]
Added comments.

**Review by achatterjee-grafana — APPROVED** [review]
LGTM

**Comment by oddlittlebird** [comment]
Thanks for contributing!

**Pull request merged**